docs/product-getting-started-guide/src/main/asciidoc/data-object-violation-create-proc.adoc

```diff
@@ -0,0 +1,79 @@
+[id='data-object-violation-proc']
+= Creating the Violation data object
+
+The following chapter instructs you how to create the Violation data object containing data fields, such as: Violation Date, Fine Amount, and Speed Limit.
+
+[float]
+== Prerequisites
+
+You have created the Driver department traffic violations project.
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver department traffic violations*.
+. Click *Create New Asset* -> *Data Object*.
+. In the *Create new Data Object* wizard, enter the following values:
+.. *Data Object*: `Violation`.
+.. *Package*: select `org.jboss.example.traffic_violations`.
+. Click *+Ok*.
```

Comment:
Find and replace all "+Ok" with just "Ok". That's what we've done to date. For consistency.

Reply:
Ha! I consistently did just the opposite. Fixing..